.github/workflows/main.yml

```diff
@@ -4,14 +4,14 @@ jobs:
   table:
     runs-on: ubuntu-latest
     name: Update README from json data
-    if: github.repository_owner == 'EddieHubCommunity'
     steps:
       - uses: actions/checkout@v2
       - name: Read/Write data into README
-        uses: eddiejaoudecommunity/gh-actions-html-table-generator@v0.1
+        uses: Panquesito7/gh-actions-html-table-generator@main
```

Comment:
I think after the pr merge in gh actions repo this should be changed to eddiehub again

Reply:
Yep, agree

Reply:
Fixed; thanks for pointing that out. 🙂